.github/workflows/push-mimir-build-image.yml

```
@@ -70,11 +70,11 @@ jobs:
         id: notification
         run: | 
           if [ ${{ steps.check_build.outputs.build }} == 'true' ]; then
-           gh pr comment $PR_URL --body "**Building new version of `mimir-build-image`**. After image is built and pushed to Docker Hub, \
-           a new commit will automatically be added to this PR with new image version `$IMAGE:$TAG`. This can take up to 1 hour."
+           gh pr comment $PR_URL --body "**Building new version of mimir-build-image**. After image is built and pushed to Docker Hub, \
+           a new commit will automatically be added to this PR with new image version \\\`$IMAGE:$TAG\\\`. This can take up to 1 hour."
           else
             echo "This PR will not trigger a build of mimir-build-image"
-            gh pr comment $PR_URL --body "**Not building new version of `mimir-build image`**. This PR modifies `mimir-build-image/Dockerfile`, but the image `$IMAGE:$TAG` already exists."
+            gh pr comment $PR_URL --body "**Not building new version of mimir-build-image**. This PR modifies mimir-build-image/Dockerfile, but the image \\\`$IMAGE:$TAG\\\` already exists."
```

[nit] This is a file path, so formatting it as monospace makes sense to me

```suggestion
            gh pr comment $PR_URL --body "**Not building new version of mimir-build-image**. This PR modifies \\\`mimir-build-image/Dockerfile\\\`, but the image \\\`$IMAGE:$TAG\\\` already exists."
```